src/infrastructure/content/ServerPort.java

```diff
@@ -1,11 +1,27 @@
 package infrastructure.content;
 
+/**
+ * 
+ * @author Badal Kumar (111701008)
+ *
+ */
+
 public class ServerPort implements IServerPort {
-	/*
+	private int port = -1;
+	/**
 	 * This method will save the port of the board server locally to send message over networking
 	 */
-	@Override
 	public void sendPort(int port) {
-		
+		this.port = port;
+	}
+	
+	/**
+	 * This method will provide the port to its caller
+	 */
+	int getPort() {
+		if (port != -1) {
+			return port;
+		}
+		return -1;
```

Comment:
Replacing with 'return port' will behave the same way

Reply:
I know. This has been done with respect to future needs (thinking of incorporating logger). I would leave it as it is for now.